components/eam/docs/user-guide/index.md

```diff
@@ -1 +1,139 @@
-start of the EAM User's Guide
+
+# EAM Users's Guide
+
+This Users's Guide describes how to set up and run EAM.
+
+## Table of contents
+
+1. [Steps to build and run EAM](#steps-to-build-and-run-eam)
+2. [Scientifically supported compsets and grids](#scientifically-supported-compsets-and-grids)
+    1. [Compsets](#compsets)
+    2. [Grids](#grids)
+3. [Customizing runs](#customizing-runs)
+    1. [Namelist changes](#namelist-changes)
+    2. [Input datasets](#input-datasets)
+    3. [Specifying output (history files)](#specifying-output-history-files)
+
+## Steps to build and run EAM
+
+A step-by-step instruction on how to run E3SM can be found [here](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/2309226536).
+
+The difference when running in atmosphere-only mode (without an interactive ocean or sea-ice) would be to change the compset and grid. Certain namelist paramters, input data files, and output file specifcations can also be modified. These are described below as ways to customize runs. 
+
+## Scientifically supported compsets and grids
+
+### Compsets
+
+All of the compsets below run with the complete set of E3SM atmospheric configuration of EAMV3. For more information on the schemes in EAMv3, see the techguide.
+
+`F2010` - Climatological present day climate (year 2010)
+
+`F1850` - Climatological pre-industrial-day climate (year 1850)
+
+`F20TR` - Historical EAM simulation with time varying sea-surface temperatures, aerosol emissions, and greenhouse gas forcings (year 1850-2014)
+
+### Grids
+
+`ne30pg2_r05_IcoswISC30E3r5` - ne30pg2 atmosphere, 0.5deg x 0.5deg land grid, and Icosahedral 30 km mesh with ice shelves cavities (wISC), E3SMv3 (E3) revision r5
+
+## Customizing runs
+
+### Namelist changes
+
+Namelist parameters can be specified in the `user_nl_eam` file. While most can be added at run time, some need to be added before `./case.setup` to have the changes applied in the simulation. 
```

Comment:
What paramateres in user_nl_eam need to be specified before case.setup?

Reply:
Talking to @wlin7, they appear to be specifically in the chemistry and RRTMG, so I've rephrased this and will note them alongside the specific parameters.

Reply:
@wlin7 I still don't understand how that works. You're saying that if I change something in user_nl_eam and then run "case.build", different code will be compiled?   I'm pretty sure CIME doesn't work that way.   The user namelist is strictly for run-time changes.